readme.md

```diff
@@ -335,9 +354,9 @@ There are a bunch of different browser events you can use in Javascript, all [li
 
 > Some programmers have qualms with W3Schools since they're mooching off the name of the W3 without actually being related to them. However, this list is accurate and easy-to-read.
 
-## Break (10 minutes / 2:10)
+## Break (4:40 - 4:50, 10 minutes)
 
-### Event Defaults (5 minutes / 2:15)
+### Event Defaults (4:50 - 4:55, 5 minutes)
 
```

Comment:
would move this above key bound events, it's probably the most important thing when it comes to the event object.